sources/packages/backend/libs/services/src/notifications/notification/notification.model.ts

```diff
@@ -176,3 +176,11 @@ export interface DailyDisbursementReportProcessingNotification {
   attachmentFileContent: string;
   fileName: string;
 }
+
+export interface SupportingUserInformationNotification {
+  givenNames: string;
+  lastName: string;
+  toAddress: string;
+  userId: number;
+  supportingUserType: string;
```

Comment:
I would ask to type the supportingUserType like this or using an enum. I believe all the options here must start with lower case.

```ts
export interface SupportingUserInformationNotification {
  givenNames: string;
  lastName: string;
  toAddress: string;
  userId: number;
  supportingUserType: "parent" | "partner" | "partners";
}
```